src/deferred.js

```diff
@@ -385,5 +394,12 @@ jQuery.extend( {
 	}
 } );
 
+function deferredTick() {
+	processQueueTimeout = undefined;
+	jQuery.each( processQueue.splice( 0 ), function( _, process ) {
```

Comment:
Is the test I added in 4bc074b correct? One notify throwing should not prevent the second...?